packages/core/strapi/lib/services/audit-logs.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const ee = require('../utils/ee');
+
+const createAuditLogsService = (strapi) => {
+  const eventListener = (name, payload = {}) => {
```

Comment:
I think we should consider a better name since this isn't an event listener but and eventListener's callback...

Reply:
saveEvent sounds good to me